packages/web-components/src/search-widget/client/components/search-form.svelte

```diff
@@ -88,32 +133,202 @@
   .search-form-input:focus {
     outline: none;
   }
+
+  .search-form-control {
+    margin-top: 8px;
+    text-align: left;
+    font-size: 1rem;
+  }
+
+  .search-form-control-radio {
+    cursor: pointer;
+    display: inline-block;
+    line-height: 1.25;
+  }
+
+  .search-form-advanced-search {
+    background: none;
+    border: none;
+    outline: none;
+    float: right;
+    cursor: pointer;
+  }
+
+  @media screen and (min-width: 560px) {
+    .filter-form-container {
+      display: flex;
+      margin-left: -0.5rem;
```

Comment:
em, not rem here please - we don't control the base font size on the HTML doc so this will not scale. Should be based on the parent font size